src/init.cpp

```diff
@@ -1161,6 +1161,7 @@ bool AppInitSanityChecks()
     // ********************************************************* Step 4: sanity checks
 
     // Initialize elliptic curve code
+    LogPrintf("Using the '%s' SHA256 implementation\n", SHA256AutoDetect());
```

Comment:
Nit: Seems this is a log message with the side-effect of detecting the SHA256 implementation. 
I'd prefer to assign the result explicitly, so that if someone happens to comment this out, or moves it to debug category, it won't just be skipped.